### DISCUSSION TITLE
List NFTs by Collection rather than Creator wallet address

### DISCUSSION THREAD

**Discussion by wxivd** [asastats/channel#31] [Ideas]
ASA Stats currently presents NFTs held in wallet by displaying the number of NFTs owned, followed by Creator wallet address. (**[91 BUAYW..R4N6M](https://www.asastats.com/BUAYW67N5MZ6XVC6PRHFHBM75COCFJINGFM3AVYDBJOGP7B4ESNPNR4N6M)**).

I would like to discuss the feasibility of presenting NFTs by Collection name. For example, The Algonautz rather than BUAYW..R4N6M. 

Points to consider:
- NFTs are easier to explore by Collection than creator wallet.
- Currently, creator's wallet address is not full and there is no Hyperlink to creator wallet.
- It is beneficial to include creator wallet address (hyperlink) in the NFT details.

**Comment by ipaleka** [comment]
@dragmz created a repository for this purpose: 

https://www.reddit.com/r/asastats/comments/th0fp1/asa_stats_v05alpha_is_online/  
> Right now NFTs are grouped by the creators under their ABCDE..FGHIJ addresses and we have used only a small part of our code for the NFT collections developed by DragMZ, a member of our Developer group. It is expected that in the future his code will be used for grouping your NFTs under the official collections they are published in.

That code still doesn't fetch all the markets, so we've used the creator address until that's completed.
